components/shared/ui/menu.tsx

```diff
@@ -182,6 +181,7 @@ export const Menu: FC<Props> = (props) => {
               <Bars3Icon className="w-6 h-6" />
             </button>
           </div>
+          <div {...createItemSmartLink(props.item.system.id)} />

```

Comment:
We have the `StandaloneSmartLinkButton` component for this purpose. Do you think it would be a good fit here?